app/app.yaml

```diff
@@ -7,11 +7,11 @@ modules:
       # there is nothing left over in the validator fee pool, so as to keep the CanWithdrawInvariant invariant.
       # NOTE: staking module is required if HistoricalEntries param > 0
       # order of emissions must come before distribution for reputers + workers take their rewards cut before validators cut
-      begin_blockers: [emissions, distribution, staking, mint]
-      end_blockers: [staking, emissions]
+      begin_blockers: [capability, emissions, mint, distribution, staking, ibc, transfer, genutil, interchainaccounts, feeibc]
+      end_blockers: [staking, ibc, transfer, capability, genutil, interchainaccounts, feeibc, emissions]
```

Comment:
Because I thought we need to mint token to distribute the reward token.

Reply:
@RedBird96 Actually, the distribution happens for the tokens minted in the last block, not for the actual block. I don't think it will change anything. 

cc @relyt29

Reply:
distribution takes the tokens that are deposited into the `fee_collector` account which mint sends to during its begin block so you'd be mixing fees from the previous block_{n-1} with minted rewards for the current block block_n

I would swap distribution and mint. I'm pretty sure the order does actually make a difference, but even if it doesn't, we know that emissions-> distribution -> mint works, I am not interested in taking chances here and screwing with the order of begin blockers

Reply:
Sure, I will fix it

Reply:
![2024-03-18-120311_752x238_scrot](https://github.com/allora-network/allora-chain/assets/4326714/ba0be30b-4142-4ed6-987d-a7b7554bf57f)
we need to preserve this order